improver_tests/acceptance/SHA256SUMS

```diff
@@ -633,73 +633,73 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
-5bfc0319d35a7b7a230fdd51f3e8b6bfddf5632b32dde8e0ff023a6b106268f7  ./wxcode-modal/spot_ties/20201209T1200Z-weather_symbols-PT01H.nc
-8f94e92886c8df8070db44b273037b032615e4b8768a3e45c708686061f0c8bc  ./wxcode-modal/spot_ties/20201209T1300Z-weather_symbols-PT01H.nc
-4fb839e28d711fd0ebbf87090358644b62f7dfa0d8dcdcad9c5dc322ce28eaac  ./wxcode-modal/spot_ties/20201209T1400Z-weather_symbols-PT01H.nc
-c798e3530eac9a45305711d5e91daa56d5005b826ce85bb48e2889e328ea965c  ./wxcode-modal/spot_ties/20201209T1500Z-weather_symbols-PT01H.nc
-83ba8460875232fc555976872cdb5ac59d5092ccc0da88f88beb9168a3403ac3  ./wxcode-modal/spot_ties/20201209T1600Z-weather_symbols-PT01H.nc
-a6698b2aae8bc223a2d362ccb3047dd9882e6c0df21962e55c577e1ad5850e29  ./wxcode-modal/spot_ties/20201209T1700Z-weather_symbols-PT01H.nc
-9290b2899eedfca038226c6707e100025ce68b0798972a60360845ccf36858d2  ./wxcode-modal/spot_ties/20201209T1800Z-weather_symbols-PT01H.nc
-ceb7106e5b8919590327a75d4a8b740fb1a2c7cb5a271a5bf980f0c1396c7d22  ./wxcode-modal/spot_ties/kgo.nc
+80c3824c422debfd717e99c076e9a53bffa368e1591dbd3a3bbf83649b015dc6  ./wxcode-modal/blend_mismatch_inputs/20201209T0700Z-weather_symbols-PT01H.nc
+a835730461e4b6cdb2b1a1986e45543d0c5b83cec6b1de5c3e761f3c2d3e3ada  ./wxcode-modal/blend_mismatch_inputs/20201209T0800Z-weather_symbols-PT01H.nc
+d68f402edd0af7b2d39998f3108da59218e547d2c18f0d661a44060e4733b2a5  ./wxcode-modal/blend_mismatch_inputs/20201209T0900Z-weather_symbols-PT01H.nc
+d8d86a1c6ef331cd453ed3e5e6e1fa322b2d9b0d2f22f65f146a4364d99a0e1a  ./wxcode-modal/blend_mismatch_inputs/20201209T1000Z-weather_symbols-PT01H.nc
+be0228790a0256bd3bdcf48fa7b6d256302d6af8583679673c5356b5397fe288  ./wxcode-modal/blend_mismatch_inputs/20201209T1100Z-weather_symbols-PT01H.nc
+cc7c6ed09a5fdbe1cc7f6b429a2d4e8063c310ddd87768b3545a6b5fbc2b0a4f  ./wxcode-modal/blend_mismatch_inputs/20201209T1200Z-weather_symbols-PT01H.nc
+78ac9e31211eb416471938ad03ee91456327e4d5a7586c35d363c7948ce93456  ./wxcode-modal/blend_mismatch_inputs/20201209T1300Z-weather_symbols-PT01H.nc
+bc559e5a54b5f05dd4ac43f08131418f4b2bfba755905a59841afc289e8ebf68  ./wxcode-modal/blend_mismatch_inputs/20201209T1400Z-weather_symbols-PT01H.nc
+b05339e9f9d5a112b57ea5f967dc559ae8b6e4874cf38ac62d8f2b8c368370f0  ./wxcode-modal/blend_mismatch_inputs/20201209T1500Z-weather_symbols-PT01H.nc
+94c46ff63155835d2745c1abdfdc6129cc8add82569d5774821036dd11c11451  ./wxcode-modal/blend_mismatch_inputs/20201209T1600Z-weather_symbols-PT01H.nc
+d0d61d34d41e901ea5799dcd4671ad309d908ef3a2a21edc53c88b9a68f3dce3  ./wxcode-modal/blend_mismatch_inputs/20201209T1700Z-weather_symbols-PT01H.nc
+88969c6413266d6353508fba8135e7b5f87faf3108becffa5e904ed731788239  ./wxcode-modal/blend_mismatch_inputs/20201209T1800Z-weather_symbols-PT01H.nc
+5e03d55eac9eaf5fa6711e7fe13dd7509e6a42ea040792588d8b3cfbdfddbd59  ./wxcode-modal/blend_mismatch_inputs/kgo.nc
+2783af44cf2c4d0e5bfe1c0c7d7dc9b346f7a86295ccedd1b74a503f9322f330  ./wxcode-modal/gridded_input/20201209T0700Z-weather_symbols-PT01H.nc
```

Comment:
Similarly to the spot files, the `title` attribute seems to have been updated from `IMPROVER Post-Processed Multi-Model Blend on UK 2 km Standard Grid` to `IMPROVER Post-Processed Multi-Model Blend Post-Processed` with this new `title` attribute being inconsistent with the suite. Would it be possible to correct this?

Reply:
Yes, these are wrong. I will change them.